GitVersionConfig.yaml

```diff
@@ -1,3 +0,0 @@
-mode: ContinuousDeployment
-assembly-versioning-scheme: MajorMinorPatch 
-#next-version: 1.0.0
```

Comment:
I'm spending a few minutes to try migrating to GitVersion. I've added the GitVersionTask nuget package and restored the yaml file. But all the versions are 0.0.0.0. I can try some more later on. I'm sure it's easy -- it's probably just the onboarding that takes a bit to figure out.